addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoDeviceHandler.java

```diff
@@ -52,7 +51,7 @@
     private RefreshStrategy refreshStrategy;
     @Nullable
     protected DEVICE device;
-    protected NAUserAdministrative userAdministrative;
+    // protected NAUserAdministrative userAdministrative;
```

Comment:
Can this be removed?

Reply:
I kept it in case we need it one day but ok I could suppress it.